consensus/hotstuff/follower_loop.go

```diff
@@ -63,7 +75,9 @@ func (fl *FollowerLoop) loop() {
 
 		select {
 		case p := <-fl.proposals:
-			err := fl.followerLogic.AddBlock(p)
+			err := fl.followerLogic.AddBlock(p.Proposal)
+			close(p.done)
```

Comment:
we will need this in event_loop.go as well

Reply:
Writing the conclusion from our meeting here. This will be added once we confirm the changes to the follower engine are working as expected since they're almost identical changes.